tests/ctst/common/common.ts

```diff
@@ -357,32 +358,23 @@ Then('kafka consumed messages should not take too much place on disk', { timeout
 
                 for (let i = 0; i < topics.length; i++) {
                     this.logger.debug('Checking topic', { topic: topics[i] });
+                    let topicCleaned = true;
                     for (let j = 0; j < newOffsets[i].partitions.length; j++) {
-                        const newMessagesAfterClean =
-                            newOffsets[i].partitions[j].low === previousOffsets[i].partitions[j].high &&
-                            newOffsets[i].partitions[j].high > previousOffsets[i].partitions[j].high;
-
-                        if (newMessagesAfterClean) {
-                            // If new messages appeared after we gathered the offsets, we need to recheck after
-                            this.logger.warn('New messages after clean', { topic: topics[i] });
-                            continue;
-                        }
+                        const partition = newOffsets[i].partitions[j];
```

Comment:
Should be renamed newOffsetPartition so that we can create a variable as well for previousOffset